Ark.Tools.ResourceWatcher.ApplicationInsights/ResourceWatcherDiagnosticListener.cs

```diff
@@ -428,14 +428,29 @@ private void _propertiesProcessContext(ISupportProperties data, ProcessContext p
             if (pc.LastState != default)
             {
                 data.Properties.Add("CheckSum_Old", pc.LastState.CheckSum);
-                data.Properties.Add("Modified_Old", (pc.LastState.ModifiedMultiple != null && pc.LastState.ModifiedMultiple.Any()) ? pc.LastState.ModifiedMultiple.Max(x => x.Value).ToString() : pc.LastState.Modified.ToString());
+                data.Properties.Add("Modified_Old", pc.LastState.Modified.ToString());
+                if (pc.LastState.ModifiedMultiple != null && pc.LastState.ModifiedMultiple.Any())
+                {
+                    foreach (var modified in pc.LastState.ModifiedMultiple)
+                    {
+                        data.Properties.Add("Modified_Old_" + modified.Key , modified.Value.ToString());
```

Comment:
I'd prefer to have all 'Old' properties with 'Old' suffix as before. Change to "Modified[$key]_Old" or similar. Same for the paired property below.